src/client/voice/BaseMediaConnection.ts

```diff
@@ -1,6 +1,7 @@
 import { streamOpts } from "../StreamOpts";
 import { VoiceOpCodes } from "./VoiceOpCodes";
 import { MediaUdp } from "./MediaUdp";
+import { normalizeVideoCodec } from "../../utils.js";
```

Comment:
Is this a typo? Should be
```suggestion
import { normalizeVideoCodec } from "../../utils";
```

Reply:
It's just a habit of mine to specify the extension, since I do most of my development with ES modules, which requires the file extension to be present. On CommonJS, either version will work fine.